packages/react-checkbox/etc/react-checkbox.api.md

```diff
@@ -4,19 +4,23 @@
 
 ```ts
 
-import { IIconProps } from '@fluentui/react-internal/lib/Icon';
+import { BaseSlots } from '@fluentui/react-compose';
+import { ClassFunction } from '@fluentui/react-compose';
 import { IRefObject } from '@fluentui/utilities';
 import { IRenderFunction } from '@fluentui/utilities';
 import { IStyle } from '@fluentui/style-utilities';
 import { IStyleFunctionOrObject } from '@fluentui/utilities';
+import { ISvgIconProps } from '@fluentui/react-icons-mdl2';
 import { ITheme } from '@fluentui/style-utilities';
 import * as React from 'react';
+import { SlotProp } from '@fluentui/react-compose';
+import { SlotProps } from '@fluentui/react-compose';
 
 // @public (undocumented)
 export const Checkbox: React.FunctionComponent<ICheckboxProps>;
 
 // @public (undocumented)
-export const CheckboxBase: React.FunctionComponent<ICheckboxProps>;
+export const CheckboxBase: import("@fluentui/react-compose").ComponentWithAs<"div", ICheckboxProps>;
```

Comment:
You need to add an explicit import of the appropriate type in the CheckboxBase file. Otherwise API Extractor will complain once we upgrade to TS 4.1.